stdlib/public/Concurrency/TaskGroup.swift

```diff
@@ -290,6 +262,74 @@ public func _unsafeInheritExecutor_withThrowingTaskGroup<ChildTaskResult, GroupR
 /// and mutation operations can't be performed
 /// from a concurrent execution context like a child task.
 ///
+/// Structured Concurrency
```

Comment:
Added a big section on this to `TaskGroup` and directing all other docs to "please read the TaskGroup docs for common behaviors"